activemodel/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Moved DecimalWithoutScale, Text, and UnsignedInteger from ActiveModel to ActiveRecord
```

Comment:
It should be Active Model and Active Record in both changelogs since we're referring to the library names and not the Ruby constants.